### PR TITLE
[fix][refactor][feat] General code overhaul

### LINKED ISSUE #18: General code cleanup
Noticed a lot of empty classes or class variables which are not applying any properties on HTML elements.
Will be combing through the code accordingly and clean it up to the best of my ability

Chores:
1. Clean up code, unwanted and unused classes
2. Reorganize CSS files and properties
3. Check content in all pages for syntax or missing content

### PR DESCRIPTION
**Pull request by Hamdrive**
This PR refers to issue #24 

**UPDATE:** All changes as per issue as been fixed, Review it at [FueledUp™ - Enrich the racer in you](https://deploy-preview-23--fueledup.netlify.app)

I will be going through all files with a fine comb and make necessary refactors or edits where necessary.
Will be updating this PR with code structure changes, which should not affect the functionality of the website. Only what's under the hood is being refactored and reorganized to follow good coding practices.

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -1,54 +1,47 @@
...
-  padding-top: var(--size-9);
-  padding-bottom: var(--size-9);
-}
-
-@media screen and (min-width: 1024px) {
+@media screen and (min-width: 1025px) {
+  /* Padding for small screens not required */
   .page-content {
     padding-left: 0;
     padding-right: 0;
```
Comment (aryan-13) [inline css/wishlist.css:22]: This can be replaced with the short hand for padding

Comment (aryan-13) [inline css/wishlist.css:22]: This can be replaced with the short hand for padding

File: index.html [hunk 2]
```diff
@@ -42,15 +42,15 @@
...
                     Signup
                   </button></a
                 >
               </li>
               <li class="nav-item">
-                <a href="pages/login.html" class=""
-                  ><button class="btn btn-md btn-login px-md">Login</button></a
+                <a href="pages/login.html"
+                  ><button class="btn btn-md btn-cta btn-login px-md">Login</button></a
                 >
```
Comment (aryan-13) [inline index.html]: The closing tags lack indentation! Code formatter can be configured.

**Review by aryan-13 — COMMENTED** [review]
Else the code looks clean!

File: pages/products.html [hunk 3]
```diff
@@ -42,15 +42,15 @@
...
                 >
               </li>
               <li class="nav-item">
-                <a href="login.html" class=""
-                  ><button class="btn btn-login btn-md px-md">Login</button></a
+                <a href="login.html"
+                  ><button class="btn btn-cta btn-login btn-md px-md">Login</button></a
                 >
               </li>
               <li class="nav-item">
```
Comment (aryan-13) [inline pages/products.html:54]: The links for wishlist and cart are not working in the product page

File: css/wishlist.css (hunk 1 above)
Comment (Hamdrive) [reply css/wishlist.css:22]: I had to be specific here as else it would remove all padding from the element

File: pages/products.html (hunk 3 above)
Comment (Hamdrive) [reply pages/products.html:54]: Fixed it! Must have been overriden during merge conflict

File: index.html (hunk 2 above)
Comment (Hamdrive) [reply index.html]: Will look into this, maybe we can discuss on call

**Review by Ronak-Mutha — APPROVED** [review]
Looks good to go ahead with changes.

File: index.html (hunk 2 above)
Comment (Hamdrive) [reply index.html]: Fixed Prettifier formatting, now looks good!

**Comment by Hamdrive** [comment]
Suggestions taken into consideration, and implemented. All reviewers approve changes. Thus this fixes #18 .
Merging and closing branch

**Pull request merged**